test/admin/e2e/general/e2e.spec.ts

```diff
@@ -358,6 +358,12 @@ describe('General', () => {
       const response = await page.goto(customLogoutRouteURL)
       expect(response.status() !== 404).toBeTruthy()
     })
+
+    test('should redirect `${adminRoute}/collections` to `${adminRoute}', async () => {
```

Comment:
This test name is misleading since the actual redirect doesn't take place when a custom view is mounted to this route. But we should test both. Maybe use globals to test for the actual redirect now.